scripts/py_matter_yamltests/matter_yamltests/parser.py

```diff
@@ -837,16 +838,32 @@ def __next__(self) -> TestStep:
         raise StopIteration
 
 
+@dataclass
+class TestParserConfig:
+    pics: str = None
+    definitions: SpecDefinitions = None
+    config_override: dict = field(default_factory=dict)
+
+
 class TestParser:
-    def __init__(self, test_file, pics_file, definitions):
+    def __init__(self, test_file: str, parser_config, TestParserConfig=TestParserConfig()):
```

Comment:
```suggestion
    def __init__(self, test_file: str, parser_config: TestParserConfig=TestParserConfig()):
```
unless I am really missing what this is trying to do.

Reply:
Obvsiouly you are right.